### PR TITLE
add interval load

### PR DESCRIPTION
**Pull request by chrfla-cognite**

### CODE REVIEW

File: frontend/src/Editor/Viewer.jsx [hunk 1]
```diff
@@ -194,11 +195,28 @@ class ViewerComponent extends React.Component {
   };
 
   runQueries = (data_queries) => {
+    let intervals = [];
```
Comment (pavelzubarev) [inline frontend/src/Editor/Viewer.jsx:198]: I would rename that variable name for calrity

File: frontend/src/_helpers/appUtils.js [hunk 2]
```diff
@@ -934,8 +934,9 @@ export function runQuery(_ref, queryId, queryName, confirmed = undefined, mode =
       [queryName]: {
         ..._ref.state.currentState.queries[queryName],
         isLoading: true,
-        data: [],
-        rawData: [],
+        // Don't reset data and rawData to empty arrays
+        // data: [],
+        // rawData: [],
```
Comment (pavelzubarev) [inline frontend/src/_helpers/appUtils.js:939]: Why do you need that? Could you explain it a bit?

Comment (chrfla-cognite) [reply frontend/src/_helpers/appUtils.js:939]: This should resolve the issue that data is reset before reloaded (when using onInterval reload). Resetting would result in two renderings of the component (one with empty data and one with reloaded data). 

The suggested approach might need some fine tuning, as this should only happen on reload, but not on initial page load.

Comment (chrfla-cognite) [reply frontend/src/_helpers/appUtils.js:939]: I'll adapt the code to cover initial page load and remove commented part afterwards.

Comment (chrfla) [reply frontend/src/_helpers/appUtils.js]: Updated to fix first load issue

File: frontend/src/Editor/Viewer.jsx (hunk 1 above)
Comment (chrfla) [reply frontend/src/Editor/Viewer.jsx]: Done ✅

**Pull request closed without merging**